src/utils/buffer.ts

```diff
@@ -271,3 +273,24 @@ export const bytesToOutputValue = (srcBuf: Buffer): [OutputValueType, Buffer] =>
 
   return [value * sign, buff];
 };
+
+/**
+ * Get the bytes from the value
+ * If value is above the maximum for 32 bits we get from 8 bytes, otherwise only 4 bytes
+ *
+ * @throws {OutputValueError} Will throw an error if output value is invalid
+ *
+ * @return {Buffer}
```

Comment:
```suggestion
 *
 * @throws {OutputValueError} Will throw an error if output value is invalid
```

The return type can be inferred from the funcion call signature.

Reply:
Removed: https://github.com/HathorNetwork/hathor-wallet-lib/pull/842/commits/c8ebab33b7ff325a1dcb38d3298ec502c8827854